mrtarget/common/__init__.py

```diff
@@ -214,3 +214,23 @@ def write(self,buf):
         self.buf = buf.strip('\r\n\t ')
     def flush(self):
         self.logger.log(self.level, self.buf)
+
+
+def str_to_int(s):
```

Comment:
These functions will swallow the exception and return zero - potentially very dangerous behaviour! I think it would be better for the caller to handle this and take appropriate action, rather than having a general function to brute force it through. If this is what is intended and designed, could you move it into the specific calling package (rather than a general function) and clear document it with the function and risky behaviour please?